UI/MinimalExtraColorPreset.cs

```diff
@@ -6,6 +6,7 @@ public class MinimalExtraColorPreset : IMinimalExtraColorPreset
         public string name { get; }
 
         public static readonly MinimalExtraColorPreset randomItem = new MinimalExtraColorPreset("Random", "random");
+        public static readonly MinimalExtraColorPreset randomUniqueItem = new MinimalExtraColorPreset("Random (Unique)", "randomUnique");
```

Comment:
I would simplify this name to just 
```suggestion
        public static readonly MinimalExtraColorPreset randomUniqueItem = new MinimalExtraColorPreset("Shuffle", "randomUnique");
```
since I think that gets the idea across better